sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -1208,6 +1211,237 @@ case class GpuToUTCTimestamp(
   }
 }
 
+class MonthsBetweenExprMeta(expr: MonthsBetween,
+                            override val conf: RapidsConf,
+                            override val parent: Option[RapidsMeta[_, _, _]],
+                            rule: DataFromReplacementRule)
+  extends ExprMeta[MonthsBetween](expr, conf, parent, rule) {
+
+  override def isTimeZoneSupported = true
+
+  override def convertToGpu(): GpuExpression = {
+    val gpuChildren = childExprs.map(_.convertToGpu())
+    assert(gpuChildren.length == 3)
+    GpuMonthsBetween(gpuChildren(0), gpuChildren(1), gpuChildren(2), expr.timeZoneId)
```

Comment:
nit: either unpack or use named params for readability
```suggestion
    val Seq(ts1, ts2, roundOff) = gpuChildren
    GpuMonthsBetween(ts1, ts2, roundOff, expr.timeZoneId)
```